extensions/objects/templates/literally-canvas-diagram-editor.component.ts

```diff
@@ -65,30 +97,202 @@ angular.module('oppia').component('literallyCanvasDiagramEditor', {
           'px X ' + maxHeight + 'px to ensure that it fits in the card.');
       };
 
-      ctrl.validate = function(data) {
-        // TODO(#9357): Will be implemented once the svg data saving
-        // functionality is implemented.
+      ctrl.isDiagramCreated = function() {
+        // This function checks if any shape has been created or not.
+        if (typeof ctrl.lc !== 'undefined' &&
```

Comment:
Any reason not to just use ctrl.lc !== undefined, and similarly elsewhere below?

Reply:
I have removed typeof. No particular reason for using typeof.